src/environmentd/src/environmentd/main.rs

```diff
@@ -544,6 +549,41 @@ pub struct Args {
         default_value = "1"
     )]
     bootstrap_builtin_analytics_cluster_replica_size: String,
+    /// The size of the builtin system cluster replicas if bootstrapping.
```

Comment:
`/// The replication factor of the builtin system cluster replicas if bootstrapping.`